packages/picasso-forms/src/RichTextEditor/RichTextEditor.tsx

```diff
@@ -34,14 +37,37 @@ export const RichTextEditor = ({ onChange, defaultValue, ...rest }: Props) => {
     [onChange, setValue]
   )
 
+  const handleLabelClick = () => {
+    if (!editorRef) {
+      return
+    }
+
+    editorRef.current?.focus()
+  }
```

Comment:
NIT: we can simplify it to a single line, because editorRef will never be `null`
```suggestion
  const handleLabelClick = () => editorRef.current?.focus()
```

Reply:
✅